client/state/themes/selectors/get-recommended-themes-filter.js

```diff
@@ -12,5 +12,7 @@ import 'calypso/state/themes/init';
  */
 export function getRecommendedThemesFilter( state, siteId ) {
 	const isUsingSiteEditor = isSiteUsingCoreSiteEditor( state, siteId );
-	return isUsingSiteEditor ? 'block-templates' : 'auto-loading-homepage';
+	return isUsingSiteEditor
+		? [ 'block-templates', 'auto-loading-homepage' ]
+		: [ 'auto-loading-homepage' ];
```

Comment:
I think this is a good place to start, and the refactoring and cleanup should be valuable!

One thing I think we need to start thinking about for the near future is being able to add the new universal themes to the recommended showcase when we roll them out for non-beta users in the coming weeks.  That way more 'classic' users are using universal themes in classic mode, making it that much easier to convert them when we want to launch FSE to all.

But this idea also gets complicated quickly as well 🤔 .  I think all block themes on dotcom are universal in that they can run the classic experience without the site editor... HOWEVER, some of the early ones still have classic counterparts (like Mayland (blocks) and Seedlet (blocks)).  Meaning whether we are adding them in beta only mode or not, we end up with both versions of the theme in the showcase (both Mayland and Mayland (blocks) for example) which both should technically provide the exact same experience in classic mode? 🤔